c++/src/CMakeLists.txt

```diff
@@ -179,15 +179,15 @@ set(SOURCE_FILES
   OrcFile.cc
   Reader.cc
   RLEv1.cc
-  RLEv2.cc
+  RleDecoderV2.cc
+  RleEncoderV2.cc
```

Comment:
The reason I prefer to Split Encoder and Decoder into two files for V2 simply because the code has grow too big that it would be very difficult to navigate if combine them into a single file.